locations/categories.py

```diff
@@ -61,6 +61,7 @@ class Categories(Enum):
     SHOP_BICYCLE = {"shop": "bicycle"}
     SHOP_BOOKMAKER = {"shop": "bookmaker"}
     SHOP_BOOKS = {"shop": "books"}
+    SHOP_BOAT_REPAIR = {"shop": "boat_repair"}
```

Comment:
Perhaps it's better as https://wiki.openstreetmap.org/wiki/Tag:shop%3Dboat and https://wiki.openstreetmap.org/wiki/Key:boat:repair ?

OSM tags for places you can buy boats and boat accessories, and places where you can receive boat repair/service features seem to be fairly vague and overlapping. For example, some of the Suzuki Marine AU locations are nominated as service centres only (`shop=boat_repair`). Other locations appear to sell all sorts of boat accessories, including outboard motors. When buying outboard motors it appears that one turns up with their boat and the shop has a workshop where they'll install the new outboard motor for you. I'm not sure it's designed such that you can rock up with a ute, pick up an outboard motor, and install it yourself/someplace else.